src/lib/pages/frontsite/PatronProfile/LoansList.js

```diff
@@ -47,4 +49,5 @@ LoansList.propTypes = {
   rowsPerPage: PropTypes.number.isRequired,
   renderListEntry: PropTypes.func.isRequired,
   noLoansCmp: PropTypes.element.isRequired,
+  renderShow: PropTypes.element,
```

Comment:
what is the reason for passing the component  as a prop? I am asking since I don't remember this part

Reply:
This way we can send an element that render the ShowAll button. We need to have this element in the LoansList, as it's placed below the pagination but before the list; but it's also better to create it in the parent elements (PatronCurrentLoans and PatronPendingLoans), so we can use the onShowAll and onShowLess functions to create it there instead of having to pass both function to the child component.